widgetastic_manageiq/__init__.py

```diff
@@ -599,6 +599,28 @@ def read(self):
         return [{key: col.text for key, col in row} for row in self]
 
 
+class ParametrizedSummaryTable(ParametrizedView):
```

Comment:
Good idea, I like it, reducing the boilerplate summary table widgets on most views.

There is a downside to this tradeoff, in that the view doesn't have an explicit implementation of the expected summary table titles.

I think its a worthy tradeoff, and that we can quickly and easily automate a test of what summary tables are available.